src/bsp/pc-rtems/src/bsp_start.c

```diff
@@ -385,7 +385,7 @@ rtems_task Init(rtems_task_argument ignored)
 /* configuration information */
```

Comment:
`* Batch mode is intended for non-interative execution.` Misspelled on line 104

Reply:
It's supposed to be "non-interactive", correct? (not non-iterative)

Reply:
@jphickey @skliper Thoughts?

Reply:
yes, it should be "non-interactive"

Reply:
Thanks Joe.